cmd/controlplane/webhooks.go

```diff
@@ -56,19 +57,22 @@ func newWebhooksServerCommand() *cobra.Command {
 				},
 			)
 			if err != nil {
-				return errors.Wrap(err, "error creating webhooks manager")
+				return errors.Wrap(err, "new manager")
```

Comment:
I don't understand the change to less descriptive error messages.

Reply:
I'm not sure how I feel about that. I think there's room to discuss it further.

In the meantime, I would say follow that style for new error messages if you like, but let's avoid changing existing ones until there's more conversation on the topic.